### PR TITLE
Implementation of Follow Request Page and Small Fixes on Follow Feature

### PR DESCRIPTION
**Pull request by laylaylo**
### **Changes:**

-  _/api/views.py_ : some small fixes on follow feature related methods
- _front_end/templates/pages/follow_requests.html_ and _front_end/views.py_ : implementation of follow request page

### **Testing?**

- The functionality has been tested manually _**at local endpoint**_ by trying different cases.

     - Accept request, Reject request, investigate how DB and follower/followed page are changed 

- Changes didn't affect old code and all unit tests for api methods are passed. But there is no auto test for front-end.

### CODE REVIEW

File: practice_app/front_end/views.py [hunk 1]
```diff
@@ -287,21 +287,36 @@ def following(request):
...
-        {'sender': 'NAME4'}
-    ]
-    context = {'page': 'Follow Requests', 'follow_requests': reqs, 'logged_in' : 1}
-    return render(request, "pages/follow_requests.html", context)
-
 
+    if request.method == "POST": 
+        button_value = request.POST.get('accept') or request.POST.get('reject')
+        if button_value:
+            action, sender_username = button_value.split('-')
```
Comment (gebenalimert) [inline practice_app/front_end/views.py]: This line causes error since button_value.split('-') also splits ORCID id.

**Review by gebenalimert — APPROVED** [review]
I pulled and tried possible scenarios and after the last commit everything seems to be working great and as intented.

**Review by uyduranomer — APPROVED** [review]
I pulled your pr and tested. Everything works as expected. Good job 👌

File: practice_app/front_end/views.py (hunk 1 above)
Comment (laylaylo) [reply practice_app/front_end/views.py]: Good catch, thanks! I fixed that.

**Pull request merged**